cluster-operator/src/test/java/io/strimzi/operator/cluster/model/EntityOperatorTest.java

```diff
@@ -541,91 +495,6 @@ public void testDefaultSecurityContext() {
         assertThat(dep.getSpec().getTemplate().getSpec().getSecurityContext(), is(nullValue()));
     }
 
-    @ParallelTest
-    public void testRestrictedSecurityContext() {
```

Comment:
I guess we should also deprecate or remove the `entityOperatorTlsSidecarContainerSecurityContext` methods in `PodSecurityContext`? Feel free to open a follow up issue for it if you don't want to do it right here in this PR.

Reply:
Ok.